### ISSUE TITLE
Add 'archive' button in production stage

### ISSUE THREAD

**Issue by heike-r** [pkp/pkp-lib#5709]
There are requests [here](https://forum.pkp.sfu.ca/t/how-to-archive-submissions-without-publishing-or-declining-them/51565) and [here ](https://forum.pkp.sfu.ca/t/how-to-archive-a-submission/33463) for a possibility to archive a submission in production stage without declining or publishing it. Scenarios where this is neccessary:
1. A journal uses OJS only for management of peer review and layout editing, but not for actual publication. However, they want to keep the accept/decline statistics so declining is not an option after the whole production is finished. Submissions are now cluttering the 'active submissions' list.
2. A journal uses OJS for management but does import the final issue/galleys per native XML. To keep the review history, the original submissions are not overwritten but kept. Now, they are cluttering the 'active submissions' list, while the imported issues are published and moved to the archive.

Is this possible with the distinction as follows:

declined (due to peer review): status=4 and stage_id=3
archived (due to editorial management practise): status=4 and stage_id=4

**Comment by asmecher** [comment]
@heike-r, would it be possible to schedule the submissions against an issue and publish the issue, but have the publication front end disabled in Setup? That way all the relevant metadata would be correct, but the contents would not be available via the publishing front end.

Just a note re: manipulating the `status` column of `submissions` directly -- starting with OJS 3.2, it's also necessary to make sure the `status` column of the `publications` table matches for `articles.current_publication_id` = `publications.publication_id`.

**Comment by NateWr** [comment]
I agree that some kind of publish-but-don't-show-it approach is the best solution here. If the submission does not get "published" in OJS, the editorial statistics will be inaccurate. 

It should be possible to get a submission into the archived list by scheduling it for publication in a future issue, but never publishing that issue. However, I think this will still not be counted as "published" in the editorial statistics.

**Comment by heike-r** [comment]
Regarding @NateWr suggestion:
A submission does not get archived, when it is assigned to an unpublished issue (in OJS 3.1.2.4). It stays in the 'active submissions' list until the issue is published.

**Comment by NateWr** [comment]
You're right @heike-r! I should have mentioned that this is part of the new `STATUS_SCHEDULED` status that was introduced in 3.2. Scheduled submissions will get moved into archived.

**Comment by rshiggin** [comment]
We are receiving complaints from users who only use OJS for editorial workflow that it's disruptive in 3.2 to have a submission moved to archive when it's assigned to a future issue. They want it to remain in their personal queue until the submission is in production, not while it's still in copyediting. There's a forum post from last year on this behavior. https://forum.pkp.sfu.ca/t/papers-scheduled-for-publishing-get-moved-to-the-archive/61759

**Comment by NateWr** [comment]
@rshiggin, editors can assign a submission to an issue but not schedule it for production. 

Steps:
1. Go to Publication > Issue
2. Click Assign
3. Assign to an unpublished issue.
4. When the publish modal pops up, click the X to not schedule it for publication.

The submission will remain in the queue. Beware, though, that when the issue is published this submission _will not be published_. When the submission is ready for publication, the editor will need to go in and schedule it for publication.

**Comment by rshiggin** [comment]
@NateWr This is super helpful, and from what I understand the d/b will be updated following these steps, but when I assign but don't schedule the submission does not appear in the future issue TOC. We're still on release 3.2.1-1. Has this been updated in a more recent release so that it appears in the TOC?

**Comment by NateWr** [comment]
No the submission will not appear in the issue TOC until it is scheduled for publication. When an issue is published, it looks for submissions assigned to the issue that are `STATUS_SCHEDULED` and only publishes those. So they wont' appear in the TOC until they've been scheduled, or else we'd run into other issues where editors click publish but don't understand why a submission wasn't published with it.

This two-step process is important, because from 3.2 there's a new pre-publication validation step. An editor can assign a submission to an issue at any time. But it can not be scheduled for publication unless it passes configured validation rules. By default, we've only got a rule that it is in the copyediting or production stages. But plugins can extend this to add pre-publication checks.

**Comment by eniocarboni** [comment]
This would be an interesting use case when never using the "Publication" tab.
Why don't you add a new status=6 in the submissions table with the "Archived" label?

**Comment by beatrizmilz** [comment]
Hi! Is there any update on this yet?

Adding an example:

Some submissions show in the "Active" tab, but they are incomplete (the authors did not finish the submission), and it's been so long - 2012 in the example - that could be archived.
![imagem](https://github.com/user-attachments/assets/75106440-3c54-449a-ba8a-b7eb71f59739)


How can we archive them?
